### ISSUE TITLE
More verbose output for `apply`

### ISSUE THREAD

**Issue by brandonkelly** [craftcms/nitro#95]
The `apply` command should output info about everything it’s doing, based on the changes it finds. At least in the case of adding new mounts, there is no mention of it.

#### Steps to reproduce

1. Add a new mount to the config:

    ```yaml
    mounts:
      - source: /Users/brandon/Sites
        dest: /nitro/test
    ```

2. Run `apply`.

    ```
    $ nitro apply
    Applied changes from /Users/brandon/.nitro/nitro-dev.yaml                       
    Editing your hosts file
    Password:
    There are no sites in the config file to remove
    ```

**Comment by jasonmccallister** [comment]
Just pushed verbose output (https://github.com/craftcms/nitro/commit/ab928e66fb9e3335df101c3e0d9105f687ea8e4e).

**Comment by jasonmccallister** [comment]
Here is an example from `add`:

```
$ nitro add plugins-dev
What should the hostname be? [plugins-dev] plugins.test
Where is the webroot? [web]
Added plugins.test to config file
Apply changes from config? [yes]
Mounting /Users/jasonmccallister/dev/plugins-dev to /nitro/sites/plugins.test
Creating symlink for plugins.test
Restarting NGINX
Applied changes from /Users/jasonmccallister/.nitro/nitro-dev.yaml
Editing your hosts file
Password: ******
```

**Comment by jasonmccallister** [comment]
Ok, I did not like the way that was implemented, so I rewrote and pushed to `develop`.

```
$ nitro add demo-site
What should the hostname be? [demo-site] 
Where is the webroot? [web] 
Added demo-site to config file
Apply changes from config? [yes] yes
Mounting ~/go/src/github.com/craftcms/nitro/demo-site to nitro-dev
Adding site demo-site to nitro-dev
Applied changes from /Users/jasonmccallister/.nitro/nitro-dev.yaml              
Editing your hosts file
Password: ******
```

**Comment by jasonmccallister** [comment]
This will be released with 1.0.0-beta.3
